src/odemis/driver/test/technolution_test.py

```diff
@@ -1517,5 +1486,73 @@ def test_get_and_subscribe(self):
         self.assertEqual(field_images[0] * field_images[1], self.counter)
         self.assertEqual(field_images[0] * field_images[1], self.counter2)
 
+    def test_error_get(self):
+        """Test that exceptions are raised if wrong settings are not caught in the wrapper, but on the ASM."""
+
+        self.MPPC.filename.value = time.strftime("testing_megafield_id-%Y-%m-%d-%H-%M-%S")
+        dataflow = self.MPPC.data
+
+        init_cellCompleteRes = self.MPPC.cellCompleteResolution.value
+        # force a value on a VA which will raise an error on the ASM
+        # use ._value in order to circumvent the setter checks
+        self.MPPC.cellCompleteResolution._value = (1100, 1100)  # max value allowed on HW is 1000; force it!
+
+        # raise exception due to wrong value in settings uploaded to ASM during the start of the acquisition
+        with self.assertRaises(Exception):
```

Comment:
In these test cases I have the feeling that the exact expected error should be specified to make it really succeed on that error only. In this case even when the connection isn't perfect on the moment of the test (or some other arbitary error) the testcase might succeed. While we actually only want the test case to succeed if the exact error we expect is raised.